easybuild/easyblocks/generic/intelbase.py

```diff
@@ -112,6 +112,7 @@ def extra_options(extra_vars=None):
         extra_vars = EasyBlock.extra_options(extra_vars)
         extra_vars.update({
             'license_activation': [ACTIVATION_LIC_SERVER, "License activation type", CUSTOM],
+            'licensed': [True, "Set to False if Intel software does not require a runtime licence", CUSTOM],
```

Comment:
I had a WIP branch where I named this `requires_license`, but I guess `requires_runtime_license` would be more accurate...